addons/hr_work_entry_contract/static/src/views/work_entry_hook.js

```diff
@@ -3,7 +3,7 @@
 import { serializeDate } from "@web/core/l10n/dates";
 import { useService } from "@web/core/utils/hooks";
 
-export function useWorkEntry({ getEmployeeIds, getRange }) {
+export function useWorkEntry({ getEmployeeIds, getRange, reload}) {
```

Comment:
You can re-use onClose as name, it's more accurate
```suggestion
export function useWorkEntry({ getEmployeeIds, getRange, onClose }) {
```

Reply:
thanks @kba-odoo 
updated